package.json

```diff
@@ -74,209 +74,209 @@
       "explorer": [
         {
           "id": "iotHubDevices",
-          "name": "Azure IoT Hub"
+          "name": "%azure-iot-toolkit.name%"
         }
       ]
     },
     "commands": [
       {
         "command": "azure-iot-toolkit.sendD2CMessage",
-        "title": "Send D2C Message to IoT Hub",
-        "category": "Azure IoT Hub"
+        "title": "%azure-iot-toolkit.sendD2CMessage.title%",
+        "category": "%azure-iot-toolkit.category%"
```

Comment:
category is not a visible section, so no need to put it for translation

Reply:
We can see category.

![image](https://user-images.githubusercontent.com/8079404/60960850-d26de880-a345-11e9-8deb-0dc396c6d157.png)